test/addons/rbd-mirror/vgrc-sample.yaml

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: replication.storage.openshift.io/v1alpha1
+kind: VolumeGroupReplicationClass
```

Comment:
What is providing the CRD for this in the drenv cluster? I do not see it here for [example](https://github.com/csi-addons/kubernetes-csi-addons/releases/download/v0.10.0/crds.yaml). What am I missing?

Reply:
I think, it comes from this kustomization: https://github.com/RamenDR/ramen/blob/c0e382d05b8fd6ffa40b987dc822b5e68d3c7780/test/addons/csi-addons/kustomization.yaml#L6

Reply:
Ack, that was the same link I used, but looks like I did not search the downloaded CRDs correctly for content. It is present in the released CRD YAML, all good.